WebAppUtils.nsh

```diff
@@ -277,12 +277,16 @@
 		File /r ${SOURCE}\Views\*
 		SetOutPath ${DEST_REAL}\Content
 		File /r ${SOURCE}\Content\*
-		SetOutPath ${DEST_REAL}\Images
-		File /r ${SOURCE}\Images\*
+		${If} ${FileExists} `${SOURCE}\Images\*.*`
```

Comment:
You can just add the ` /nonfatal` switch to `File` and it won't error out.

http://webcache.googleusercontent.com/search?q=cache:c1KDAi67c8oJ:nsis.sourceforge.net/Reference/File+&cd=2&hl=en&ct=clnk&gl=us